Targets/Mashow/Sources/DrinkSelection/View/DrinkSelectionViewController.swift

```diff
@@ -0,0 +1,287 @@
+//
+//  DrinkSelectionViewController.swift
+//  Mashow
+//
+//  Created by ZENA on 8/3/24.
+//  Copyright © 2024 com.alcoholers. All rights reserved.
+//
+
+import UIKit
+import Combine
+import SnapKit
+
+final class DrinkSelectionViewController: UIViewController {
+    
+    init(viewModel: DrinkSelectionViewModel!) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    let viewModel: DrinkSelectionViewModel!
```

Comment:
```suggestion
    let viewModel: DrinkSelectionViewModel
```
init으로 받은 거라서 `!` 없어도 될듯요?? 전체적으로 수정 한 번 해주십셔ㅎ
(난 init으로 안 했는데 init으로 고쳐놓겠음. 나도 이게 더 좋아서)